pkg/sethandler/controller.go

```diff
@@ -290,21 +303,20 @@ func getInfraContainerPath(podStatus v1.PodStatus, searchPath string) string {
 	return pathToInfraContainer
 }
 
-func (setHandler *SetHandler) applyCpusetToInfraContainer(pod v1.Pod, pathToSearchContainer string) error {
+func (setHandler *SetHandler) applyCpusetToInfraContainer(podStatus v1.PodStatus, pathToSearchContainer string) error {
 	cpuset := setHandler.poolConfig.SelectPool(types.DefaultPoolID).CPUs
 	if cpuset.IsEmpty() {
 		//Nothing to set. We will leave the container running on the Kubernetes provisioned default cpuset
-		log.Println("WARNING: for some reason DEFAULT cpuset was quite empty. Cannot adjust cpuset for infra container of pod: " + string(pod.ObjectMeta.UID))
+		log.Println("WARNING: for some reason DEFAULT cpuset was quite empty. Cannot adjust cpuset for infra container")
```

Comment:
these logs are entirely useless in production without an object identifiers
either Pod name + Pod ns, or Poid UID must be included in them to have any value